app/src/main/java/com/academy/ramon/marvelcomicviewer/MainActivity.java

```diff
@@ -24,14 +26,17 @@
 public class MainActivity extends Activity {
     @BindView(R.id.rvHeroes) RecyclerView rvHeroes;
     List<Heroes> heroes;
+    Context context;
 
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-        ButterKnife.bind(this);
         getHeroesList();
+        context = this;
```

Comment:
The thing is I was trying to do all this in a utils class in order to leave all this to an independent class, when I moved it here I forgot to remove it haha sorry